openfe/protocols/openmm_rfe/equil_rfe_methods.py

```diff
@@ -823,6 +825,30 @@ def run(self, *, dry=False, verbose=True,
         else:
             return {'debug': {'sampler': sampler}}
 
+    @staticmethod
+    def analyse(where) -> dict:
+        # don't put energy analysis in here, it uses the open file reporter
+        # whereas structural stuff requires that the file handle is closed
+        ret = subprocess.run(['openfe_analysis', str(where)],
+                             stdout=subprocess.PIPE,
+                             stderr=subprocess.PIPE)
+        if ret.returncode:
+            return {'error': ret.stderr}
```

Comment:
because it'll then get squashed into the outputs dictionary, maybe something more verbose like "structure_analysis_error" would be better here?